os/board/rtl8730e/src/component/bluetooth/driver/platform/amebad2/hci/hci_platform.c

```diff
@@ -17,7 +17,11 @@
 #include "spi_api.h"
 
 /* Corresponds to LD external flash starting address */
-#define EXT_FLASH_BASE_ADDRESS   			0x09000000
+#if CONFIG_RTL8730E_BOARD_REVISION >= 7
+#define EXT_FLASH_BASE_ADDRESS				0x0A000000
+#else
+#define EXT_FLASH_BASE_ADDRESS				0x09000000
+#endif
```

Comment:
If we hard-coded here, do we need the config - CONFIG_EXTERNAL_FLASH_START_ADDR?
If we have this kind of code at several locations, I think we can easily miss changing with new configuration. I think we should have some specific file to have changeable values. It can be a Kconfig.

Reply:
This address will be remove in https://github.com/Samsung/TizenRT/pull/6303.

Reply:
@ziyik Ok. Let's merge this after resolving the conflict.
Could you check that?

Reply:
I have rebased my PR, please check and verify